src/drivers/gps/gps.cpp

```diff
@@ -381,7 +404,10 @@ GPS::task_main()
 					_report_gps_pos.epv = 10000.0f;
 					_report_gps_pos.fix_type = 0;
 
-					if (!(_pub_blocked)) {
+					/* update mavlink hil gps parameter */
+					update_param_hil_gps();
```

Comment:
@mhkabir  

Thanks, However this code already calls param_find() just only once. right?  :-)
Anyway, I will move  the param_find() to init function.

+	if (_param_use_hil_gps == PARAM_INVALID) {
+		_param_use_hil_gps = param_find("MAV_USEHILGPS");
+	}